Cargo.toml

```diff
@@ -95,8 +96,9 @@ fontconfig-dlopen = ["font-kit/source-fontconfig-dlopen"]
 
 # Misc
 datetime = ["chrono"]
-evcxr = ["svg_backend"]
+evcxr = ["svg_backend", "bitmap_backend", "image"]
 deprecated_items = [] # Keep some of the deprecated items for backward compatibility
+evcxr_bitmap_figure = ["bitmap_backend", "evcxr"]
```

Comment:
And I think want we could do is instead of adding a new feature option, we can use the conditional compilation in the code. I mean if user enables both evcxr and bitmap_backend and then the code for BitMapWrapper will be compiled